presto-main/src/main/resources/webapp/assets/query.js

```diff
@@ -892,6 +892,32 @@ var QueryDetail = React.createClass({
     },
     renderFailureInfo: function() {
         var query = this.state.query;
+        var taskRow = "";
+        var hostRow = "";
+        if (query.failureTask != null) {
+            taskRow = (
+            <tr>
+                <td className="info-title">
+                    Failed Task
+                </td>
+                <td className="info-text">
+                        { query.failureTask }
```

Comment:
If the task list is available, make this a link to the task, just like it is in the task table below.